src/REAXFF/fix_acks2_reaxff.cpp

```diff
@@ -396,6 +396,34 @@ void FixACKS2ReaxFF::init_matvec()
     }
   }
 
+  /* compute condition number of Jacobi preconditioner */
+  double sing_max = std::numeric_limits<double>::min();
```

Comment:
To use these function templates, `#include <limits>` should be added in the beginning of the file.